lib/openapi3.js

```diff
@@ -104,8 +124,7 @@ function fakeProdCons(data) {
                     let schema = op.requestBody.content[rb].schema;
                     if (schema['x-widdershins-oldRef']) {
                         data.bodyParameter.refName = schema['x-widdershins-oldRef'].replace('#/components/schemas/', '');
-                    }
-                    else {
```

Comment:
This is the first of many unrelated formatting changes even with GitHub diff set to ignore whitespace changes. Could we lose these please?